code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -131,8 +131,11 @@
 
 /////////////////////////GAS OVERLAYS//////////////////////////////
 
+//NSV13change - moves this proc up to /turf due to auxmos sometimes calling it on closed turfs and runtiming.
+/turf/proc/update_visuals()
+	return	//Avoid issues from closed turfs getting turf visuals updated (?? why is it even doing that)
```

Comment:
auxmos shouldn't be updating visuals on closed turfs. If it is somehow, this is just fixing the symptoms not the root problem (and it's associated performance burden). Bee has had it on open only forever and it works fine

Reply:
Would you happen to have the runtime log of update_visuals being called on closed turfs?

Reply:
Auxmos callbacks, only thing I got was
![image](https://user-images.githubusercontent.com/46569814/147028921-3339dcd7-51ed-4587-9058-6f73d3f97d75.png)

Reply:
(aware this only fixes the symptom, I just have absolutely zero idea what specifically causes it to also target /closed turfs)